static-analysis/src/test/java/com/example/static_analysis/OptionalArgumentRequiredDetectorTest.kt

```diff
@@ -0,0 +1,59 @@
+package com.example.static_analysis
+
+import com.android.tools.lint.checks.infrastructure.LintDetectorTest
+import com.android.tools.lint.detector.api.Detector
+import com.android.tools.lint.detector.api.Issue
+import com.example.static_analysis.detectors.OptionalArgumentRequiredDetector
+import java.io.File
+import java.nio.file.Files
+import java.nio.file.Paths
+
+class OptionalArgumentRequiredDetectorTest : LintDetectorTest() {
+    private val argumentDependencies =
+        getContent(TEST_DIR, "ViewModelArgumentDependencies.kt")
+
+    fun testViewModelArgumentsNoWarnings() {
+        val noErrorsTest = getContent(TEST_DIR, "ViewModelOptionalArgumentTestNoWarnings.kt")
+
+        lint().files(
+            kotlin(argumentDependencies),
+            kotlin(noErrorsTest)
+        ).allowCompilationErrors(false)
+            .run()
+            .expect("No warnings.")
+    }
+
+    fun testViewModelArgumentsShowWarnings() {
+        val testWithErrors =
+            getContent(TEST_DIR, filename = "ViewModelOptionalArgumentTestShowsError.kt")
+
+        lint().files(
+            kotlin(argumentDependencies),
+            kotlin(testWithErrors)
+        ).allowCompilationErrors(false)
+            .run()
+            .expect(
+                """
+                src/com/linecorp/lich/viewmodel/ArgumentTest.kt:5: Error: Optional argument initialized as required. [LichArgumentInitializationInconsistency]
```

Comment:
`LichArgumentInitializationInconsistency` should be `ArgumentInitializationInconsistency`?

Reply:
Good catch! thanks